src/go/rpk/pkg/cli/debug/bundle/bundle.go

```diff
@@ -160,7 +161,7 @@ func NewCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 	f.StringVarP(&outFile, outputFlag, "o", "", "The file path where the debug file will be written (default ./<timestamp>-bundle.zip)")
 	f.DurationVar(&timeout, "timeout", 31*time.Second, "How long to wait for child commands to execute (e.g. 30s, 1.5m)")
 	f.DurationVar(&metricsInterval, "metrics-interval", 10*time.Second, "Interval between metrics snapshots (e.g. 30s, 1.5m)")
-	f.StringVar(&logsSince, "logs-since", "", "Include log entries on or newer than the specified date (journalctl date format, e.g. YYYY-MM-DD")
+	f.StringVar(&logsSince, "logs-since", "yesterday", "Include log entries on or newer than the specified date (journalctl date format, e.g. YYYY-MM-DD")
```

Comment:
```suggestion
	f.StringVar(&logsSince, "logs-since", "yesterday", "Include log entries on or newer than the specified date (journalctl date format, e.g. YYYY-MM-DD"). Default: "yesterday"
```

Reply:
Yes, this is what we mean by `journalctl date format`, a standard defined in [systemd.time](https://www.freedesktop.org/software/systemd/man/latest/systemd.time.html). I added "yesterday/now" to the help text

The default value is included by cobra when printing the help text.

Now it looks:
```
      --logs-since string                   Include log entries on or newer than the specified date (journalctl date format, e.g.
                                            YYYY-MM-DD or today/yesterday (default "yesterday")
```

Reply:
Perfect! thanks